org.aimodels.aiy.json

```diff
@@ -0,0 +1,45 @@
+{
+  "app-id": "org.aimodels.aiy",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "22.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "aiy",
+  "finish-args": [
+    "--share=network"
+  ],
+  "modules": [
+    {
+      "name": "aiy",
+      "buildsystem": "simple",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/visioninit/aiy.git",
+          "branch": "main"
+        }
+      ],
+      "modules": [
+        "resources/build/flatpak/pypi-dependencies.json"
```

Comment:
this file is missing.

Reply:
Hi, I think I'm going to have to figure out how to take this off flathub sometime. After I was informed that command line applications aren't a good match for flathub I didn't really keep going with it here.